docs/technologies.md

```diff
@@ -61,20 +59,6 @@ The following line is the key:
 options.view_as(SetupOptions).setup_file = str(setup_file_path)
 ```
 
-If you want to provide requirements for your Beam process, you can do it through the
```

Comment:
I think you can mention that we don't recommend using `requirements_file` because all the dependencies should be placed inside the setup.py, through resources/requirements.txt.

Reply:
I made this feature "hidden" for purpose.
If package added via 'beam..requirements_file` but not mentioned in project `requirements.in` it will be available only in production, making local development and writing tests more complicated...  For inexperienced users it might complicated to understand / debug.
Also advanced users might use this `requirements_file` anyway, even without mentioning in documentation (because they have 'beam' own docs).